splice_spark2/src/main/spark2.3/org/apache/spark/sql/execution/datasources/jdbc/SpliceRelation2.scala

```diff
@@ -0,0 +1,68 @@
+package org.apache.spark.sql.execution.datasources.jdbc
```

Comment:
Copyright?  Not a blocker for bringing this in, but should be added at some point.